### ISSUE TITLE
Create Movie - Waterfall (and impact on Archipels)

### ISSUE THREAD

**Issue by 2amarkina** [blockframes/blockframes#9327]
### Context
For the creation of the Movie for the Archipels we are using the Movie Tunnel (user leaves the main layout and accesses the dedicated layout that is changing accordingly to the production status). 

For the Blockframes projet we aim to keep the user inside the main layout and simplify the form as much as we can. 
It might be possible that the movie creation will need to have several steps, but for the moment we will stick to the strict necessary info. 

Also, we would like to see if we are able to apply such kind of logic to the Archipels. That means to have a page with the mandatory info inside the main layout and propose to fill in more details in extra-layout lately. 

Moreover, the actual problem on the Archipels is that we create a Movie directly after clicking on "Add Title". That leads to the fact that there are many empty titles created. We would like to change this with the upcoming main layout mandatory page. 

### Goals
- [x] Integrate the screen of Create Movie ; 
- [x] Movie should be created only after clicking on the Create Movie CTA ; 
- [x] The arrow back should lead to the previous page (Empty Dashboard or List of Movies accordingly) ; 
- [x] If user has started fill in the informations about his Movie but clicks to any link that makes them leave the page we need to show the alert modal ; 
- [ ] See if it is possible to introduce the same logic (create a movie after saving and not at the moment of adding) for the Archipels ; 
- [ ] See if it is possible to introduce the same layout for the Archipels (all the mandatory fields on the main-app layout without movie tunnel) ; 
- [ ] See if the Movie Tunnel on the Archipels can be introduced after clicking on "More details". 

### Ressources
[Figma link to Create Movie](https://www.figma.com/file/s0qk9gGKVSJl8zXTPuNipr/UI---Blockframes?type=design&node-id=164-25855&t=qrGNnKMHPuEWFPG4-4)

[Figma link to the Alert Modal ](https://www.figma.com/file/s0qk9gGKVSJl8zXTPuNipr/UI---Blockframes?type=design&node-id=164-25491&t=qrGNnKMHPuEWFPG4-4)

[Figma link to the Create Movie with opened list of roles](https://www.figma.com/file/s0qk9gGKVSJl8zXTPuNipr/UI---Blockframes?type=design&node-id=164-25697&t=qrGNnKMHPuEWFPG4-4)

**Comment by phpgeekfr** [comment]
@2amarkina last 3 unchecked checkboxes should be moved to new issue

**Comment by 2amarkina** [comment]
[Created here ](https://app.zenhub.com/workspaces/blockframes-60b0a31838e485001b3a619f/issues/gh/blockframes/blockframes/9394 )
